packages/lab/src/system-status/SystemStatus.css

```diff
@@ -0,0 +1,48 @@
+/* Styles applied to root component */
+.saltSystemStatus {
+  --saltIcon-color: var(--salt-content-bold-foreground);
+
+  background: var(--saltSystemStatus-background, var(--systemStatus-background));
+  border-color: var(--saltSystemStatus-borderColor, var(--systemStatus-borderColor));
+  border-width: var(--saltSystemStatus-borderWidth, var(--salt-size-border));
+  border-style: var(--saltSystemStatus-borderStyle, var(--salt-container-borderStyle));
+  box-sizing: border-box;
+  display: inline-flex;
```

Comment:
Worth to think whether inline or block makes more sense

Reply:
Following the same CSS as that of the Banner implementation.
Also, inline makes more sense for our use case since we want to have the status icon, the content's first line and the action to sit in same row and align themselves.